### DISCUSSION TITLE
Reading list section on home page

### DISCUSSION THREAD

**Discussion by PleaseStopAsking** [LeoKlaus/plappa#11] [Feature Requests]
I have a habit of getting a new book and forgetting about it once it gets shuffled into my library. It would be nice to have the ability to add an item to a `Reading List` section which would allow users to keep track of books they want to read next.

ABS nor Jellyfin have this functionality so it would have to be a list that is maintained within the app itself.

**Comment by LeoKlaus** [comment]
I think this is a great idea and shouldn't be hard to implement. I'll let you know once I have something ready!

**Comment by LeoKlaus** [comment]
I've just released an update to TestFlight (build 20) that includes a reading list! The reading list is synced via iCloud regardless of server type.
